catalog/video/catalog-mpegts-stream/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -52,6 +52,11 @@
                 name="Metacard Update Initial Delay" id="metacardUpdateInitialDelay" required="false"
                 type="Long" default="2" />
 
+        <AD
+                description="Distance tolerance used to simplify WKT data when merging video chunk metadata with the parent metacard. All vertices in the simplified geometry will be within this distance of the original geometry. The tolerance value must be non-negative."
```

Comment:
I added the units, but I'm not sure how/if the description should be simplified. @jlcsmith @jaymcnallie